contracts/colony/ColonyRoles.sol

```diff
@@ -86,26 +86,24 @@ contract ColonyRoles is ColonyStorage {
     uint256 _childSkillIndex,
     address _user,
     uint256 _domainId,
-    bytes32 _roles,
-    bool _setTo
+    bytes32 _roles
   ) public stoppable authDomain(_permissionDomainId, _childSkillIndex, _domainId) archSubdomain(_permissionDomainId, _domainId)
   {
     // This is not strictly necessary, since these roles are never used in subdomains
     require(_roles & ROOT_ROLES == 0 || _domainId == 1, "colony-bad-domain-for-role");
-
     require(uint256(_roles) < 2 ** uint256(ColonyRole.NUMBER_OF_ROLES), "colony-roles-do-not-exist");
```

Comment:
Is this still necessary?

Reply:
No it is not, although I think it's worth keeping the test, which I can rework.